test/lib/pgBackRestTest/Common/Host/HostDbCommonTest.pm

```diff
@@ -202,17 +194,11 @@ sub configRecovery
 
     if (@stryRecoveryOption)
     {
-        $oConfig{$strStanza}{&OPTION_RESTORE_RECOVERY_OPTION} = \@stryRecoveryOption;
+        $oConfig->{$strStanza}{&OPTION_RESTORE_RECOVERY_OPTION} = \@stryRecoveryOption;
     }
 
     # Save db config file
-    iniSave($self->backrestConfig(), \%oConfig, true);
-
-    # Save backup config file
-    if ($oHostBackup->nameTest(HOST_BACKUP))
-    {
-        iniSave($oHostBackup->backrestConfig(), \%oRemoteConfig, true);
```

Comment:
Since there is only one fileStringWrite I'm guessing you discovered you did not need one for $oHostBackup->backrestConfig().